groups/chd-infrastructure/locals.tf

```
@@ -50,6 +50,18 @@ locals {
     }
   ] : []
 
+  # Define the NLB FTP target group ARNs (index 0) for ASG registration
+  chd_fe_internal_ftp_target_group_arn = [module.nlb_fe_internal.target_group_arns[0]]
+  chd_fe_external_ftp_target_group_arn = [module.nlb_fe_external.target_group_arns[0]]
+
+  # Generate listener configuration for FTP passive ports
+  chd_fe_ftp_passive_listeners = [
+    for num in range(var.fe_ftp_passive_ports_start, var.fe_ftp_passive_ports_end) : {
+      port               = format("%s%d", "", num)
```

Can we simplify the expression here by removing the empty string value and `%s` verb:

```suggestion
      port               = format("%d", num)
```